src/components/post-test-summary-section/post-test-summary-section.html

```diff
@@ -0,0 +1,14 @@
+<ion-card class="mes-test-summary-section-card" [ngClass]="{'hidden': summary.total === 0}">
```

Comment:
it's good but just fyi we can hide/show elements using `[hidden]="experssion" ` too :)